src/sdk/data/data.service.ts

```diff
@@ -466,5 +466,102 @@ export class DataService extends Service {
 
     return result ? result : null;
   }
-}
 
+  async getTokenLists(): Promise<TokenList[]> {
+    const { apiService } = this.services;
+
+    const { result } = await apiService.query<{
+      result: TokenLists;
+    }>(
+      gql`
+        query($chainId: Int) {
+          result: tokenLists(chainId: $chainId) {
+            items {
+              name
+              endpoint
+              isDefault
+              createdAt
+              updatedAt
+            }
+          }
+        }
+      `,
+      {
+        models: {
+          result: TokenLists,
+        },
+      },
+    );
+
+    return result.items;
```

Comment:
We're assuming items is always available here, what happens if this `await` errors?

Reply:
I have changed in code as if await fails or we don't get a result then it will return an empty array here.